Cargo.toml

```diff
@@ -68,3 +68,4 @@ arrow-odbc = { version = "9.0.0" }
 snowflake-api = { git = "https://github.com/spiceai/snowflake-rs.git", folder = "snowflake-api", rev = "5dab964abef23314e7a3f27ae89b83352e315206" }
 suppaftp = { version = "5.3.1", features = ["async"] }
 datafusion-federation = { git = "https://github.com/phillipleblanc/datafusion-federation.git", branch = "spiceai" }
+datafusion-federation-sql = { git = "https://github.com/phillipleblanc/datafusion-federation.git", folder = "sources/sql", branch = "spiceai" }
```

Comment:
Once I upstream all of the changes I need to make, I will switch this over to use the `datafusion-contrib` org.